lbjava/pom.xml

```diff
@@ -29,7 +29,6 @@
 			<groupId>gurobi</groupId>
 			<artifactId>gurobi</artifactId>
 			<version>5.0.1</version>
-			<optional>true</optional>
```

Comment:
This was causing the gurobi/GRBException somehow. The error seemed to be during some code-generation step in LBJava. If we don't see the exception at other places, we can not have this change.